### PR TITLE
Complete first task.

### PR DESCRIPTION
**Pull request by Jasurbekdeveloper**
This is a PR created for first task of the homework.

### CODE REVIEW

File: GithubDemo/GithubDemo/Program.cs [hunk 1]
```diff
@@ -4,9 +4,15 @@ internal class Program
...
+
+            if (inputNumber >= 0)
             {
-                Console.WriteLine(i);
+                Console.WriteLine("Positive!");
+            }
+            else
+            {
+                Console.WriteLine("Negative");
             }
```
Comment (Boburm1rzo) [inline GithubDemo/GithubDemo/Program.cs]: You can remove brackets.

File: GithubDemo/GithubDemo/Program.cs [hunk 2]
```diff
@@ -4,9 +4,15 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            for (int i = 0; i < 100; i++)
+            int inputNumber = int.Parse(Console.ReadLine());
```
Comment (Boburm1rzo) [inline GithubDemo/GithubDemo/Program.cs]: Use int.TryParse instead of int.Parse

Comment (Jasurbekdeveloper) [reply GithubDemo/GithubDemo/Program.cs]: Done.

File: GithubDemo/GithubDemo/Program.cs (hunk 1 above)
Comment (Jasurbekdeveloper) [reply GithubDemo/GithubDemo/Program.cs]: I think we should always use brackets.

**Pull request merged**